website/common/locales/en/messages.json

```diff
@@ -31,6 +31,7 @@
   "messageAlreadyPurchasedGear": "You purchased this gear in the past, but do not currently own it. You can buy it again in the rewards column on the tasks page.",
   "messageAlreadyOwnGear": "You already own this item. Equip it by going to the equipment page.",
   "messageHealthAlreadyMax": "You already have maximum health.",
+  "messageHealthAlreadyMin": "You already have the minimum health.",
```

Comment:
Can you make this: "Oh no! You have already run out of health so it's too late to buy a health potion, but don't worry - you can revive!"